simplecpp.h

```diff
@@ -36,6 +36,7 @@
 #  else
 #    define SIMPLECPP_LIB
 #  endif
+#  pragma warning(disable: 4706)
```

Comment:
Perhaps this could be put in its own block of preprocessor directives?
Similar to how `nullptr` is defined as `NULL` in this file.